submissions/burning-questions/burning-questions.md

```diff
@@ -0,0 +1,367 @@
+
+
+
+{% include image.html
+
+src= 'src'
+ alt='Module effects on power consumption'
+ caption='Module effects on power consumption'
+
+%}
+
+Now let's demonstrate the use of modules. First, let's look at the case of efficiency in coal liquification with heavy oil cracking. Refineries and chemical plants have 3 module slots, so we can place 3 efficiency 1 modules to reduce power consumptions by 80% for each: 
+
+{% include image.html
+
+src= 'src'
+ alt='Processing coal into solid fuel with heavy oil cracking, at minimum power consumption'
+ caption='Processing coal into solid fuel with heavy oil cracking, at minimum power consumption'
+
+%}
+
+The total power consumption drops from about 3.28MJ to about 0.66MJ, which translates to the energy gains of 116%, compared to the default case of 103%. This increase is nice but it is not that large because processing energy was already relatively small. Now let us look at the same example with 3 productivity 3 modules instead to see the effects of maximum productivity:
+
+{% include image.html
+
+src= 'src'
+ alt='Processing coal into solid fuel with heavy oil cracking, at maximum productivity'
+ caption='Processing coal into solid fuel with heavy oil cracking, at maximum productivity'
+
+%}
+
+We see that with the cumulative effects of 30% extra product output across the process results in more than double the amount of solid fuel as before! This adds up to an extra energy gain of nearly 100MJ! However, we also see that total power consumption has greatly increased to about 50.6MJ. Nevertheless, our overall energy gains have become `185.6 - 50.6 - 40 - 1.5 = 93.5MJ`, which is about 134% of energy gains instead of the default case of 103%. Meanwhile, our machines are working at half the speed as before, which makes power get consumed for twice as long.
+
+Hence finally, let's try out the compensating effects of speed modules by switching to 2 productivity 3 modules and 1 speed 3 module each:
+
+{% include image.html
+
+src= 'src'
+ alt='Processing coal into solid fuel with heavy oil cracking, at high productivity + speed'
+ caption='Processing coal into solid fuel with heavy oil cracking, , at high productivity + speed'
+
+%}
+
+We now find numbers somewhere in the middle of the previous two cases. We have about 12.2 solid fuel now, which is a good gain and it gives about 146.4MJ. We see that the total power consumption is now about 20.3MJ. Now, our overall energy gains have become `146.4 - 20.3 - 40 - 1.5 = 84.6MJ`, which is about 112% of energy gains instead of the default case of 103%. We discover that full productivity performs better in this case despite the machines slowing down so much.
```

Comment:
```suggestion
We now find numbers somewhere in the middle of the previous two cases. We have about 12.2 solid fuel, which is a good gain giving about 146.4MJ. We see that the total power consumption is now about 20.3MJ. Now, our overall energy gains have become `146.4 - 20.3 - 40 - 1.5 = 84.6MJ`, which is about 112% of energy gains, instead of the default case of 103%. We discover that full productivity performs better in this case, despite the machines slowing down so much.
```